.github/workflows/cicd.yml

```diff
@@ -75,14 +76,15 @@ jobs:
         if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release-') }}
         run: |
           export VERSION=$(cat VERSION)
+          export BRANCH=$(cat BRANCH)
           git pull --rebase
           make generate-operator-yaml
           git config --global user.name ${{ secrets.ACTIONS_USER }}
           git config --global user.email ${{ secrets.ACTIONS_EMAIL }}
           git diff astraconnector_operator.yaml
           git add astraconnector_operator.yaml
           git commit -m "Auto updating the operator image tag" -n
-          git push --set-upstream origin main
+          git push --set-upstream origin $BRANCH
```

Comment:
Also noticed this wouldn't have worked once we cut a release branch, so fixed this too